README.md

```diff
@@ -208,181 +181,26 @@ The has_ancestry method supports the following options:
     :depth_cache_column    Pass in a symbol to store depth cache in a different column
     :primary_key_format    Supply a regular expression that matches the format of your primary key
                            By default, primary keys only match integers ([0-9]+)
-    :touch                 Instruct Ancestry to touch the ancestors of a node when it changes, to
-                           invalidate nested key-based caches. (default: false)
-
-# (Named) Scopes
-
-Where possible, the navigation methods return scopes instead of records. This
-means additional ordering, conditions, limits, etc. can be applied and that
-the result can be either retrieved, counted, or checked for existence. For
-example:
-
-```ruby
-node.children.where(:name => 'Mary').exists?
-node.subtree.order(:name).limit(10).each { ... }
-node.descendants.count
-```
-
-For convenience, a couple of named scopes are included at the class level:
-
-    roots                   Root nodes
-    ancestors_of(node)      Ancestors of node, node can be either a record or an id
-    children_of(node)       Children of node, node can be either a record or an id
-    descendants_of(node)    Descendants of node, node can be either a record or an id
-    indirects_of(node)      Indirect children of node, node can be either a record or an id
-    subtree_of(node)        Subtree of node, node can be either a record or an id
-    siblings_of(node)       Siblings of node, node can be either a record or an id
-
-Thanks to some convenient rails magic, it is even possible to create nodes
-through the children and siblings scopes:
-
-    node.children.create
-    node.siblings.create!
-    TestNode.children_of(node_id).new
-    TestNode.siblings_of(node_id).create
-
-# Selecting nodes by depth
-
-When depth caching is enabled (see has_ancestry options), five more named
-scopes can be used to select nodes on their depth:
-
-    before_depth(depth)     Return nodes that are less deep than depth (node.depth < depth)
-    to_depth(depth)         Return nodes up to a certain depth (node.depth <= depth)
-    at_depth(depth)         Return nodes that are at depth (node.depth == depth)
-    from_depth(depth)       Return nodes starting from a certain depth (node.depth >= depth)
-    after_depth(depth)      Return nodes that are deeper than depth (node.depth > depth)
-
-The depth scopes are also available through calls to descendants,
-descendant_ids, subtree, subtree_ids, path and ancestors. In this case, depth
-values are interpreted relatively. Some examples:
```

Comment:
some of the values below are the same as the ones above.

lets deprecate the hash key to_depth (different pr?) it doesn't work as an association. it probably would have worked in rails 3 land, but we're not there anymore.

the ones below just look like examples. not sure if that has value